colors.json

```diff
@@ -466,18 +466,18 @@
   "ui-red-pressed": "#7c1d13",
   "ui-background": "#f3f3f3",
   "ui-foreground": "#353535",
```

Comment:
Are these correct? Isn't in "light theme" ui-foreground: `$blk-000` (#fff) and ui-background: `$blk-005` (#f8f8f8)

Reply:
correct... same issue. brought this up with @julio8a too.

These were already there... so existing